gnd/src/main/java/com/google/android/gnd/repository/DataRepository.java

```diff
@@ -249,13 +258,21 @@ public Completable saveFeature(Feature feature) {
         .andThen(dataSyncWorkManager.enqueueSyncWorker(feature.getId()));
   }
 
+  private Maybe<Project> loadLastActiveProject() {
+    if (isOfflineModeEnabled()) {
+      return localDataStore.getLastActiveProject();
```

Comment:
As mentioned, can we just use the id in the localValueStore and fetch project by id?

Reply:
Done